### PR TITLE
web/satellite: saving selected project in local storage implemented

### PR DESCRIPTION
**Pull request by VitaliiShpital**
What: now reloading page will not select first project in stored projects array. Lastly selected one will be used

Why: enhancements

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/header/projectSelection/ProjectSelectionDropdown.vue [hunk 1]
```diff
@@ -42,6 +42,7 @@ export default class ProjectSelectionDropdown extends Vue {
 
     public async onProjectSelected(projectID: string): Promise<void> {
         this.$store.dispatch(PROJECTS_ACTIONS.SELECT, projectID);
+        localStorage.setItem('localProjectKey', JSON.stringify(this.$store.getters.selectedProject));
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/header/projectSelection/ProjectSelectionDropdown.vue]: save only selected project id

File: web/satellite/src/components/header/projectSelection/ProjectSelectionDropdown.vue [hunk 2]
```diff
@@ -42,6 +42,7 @@ export default class ProjectSelectionDropdown extends Vue {
 
     public async onProjectSelected(projectID: string): Promise<void> {
         this.$store.dispatch(PROJECTS_ACTIONS.SELECT, projectID);
+        localStorage.setItem('selectedProjectId', this.$store.getters.selectedProject.id);
```
Comment (crawter) [inline web/satellite/src/components/header/projectSelection/ProjectSelectionDropdown.vue]: lets wrap all localstorage calls in some programming module, not to populate a lot of constants and hardcoded strings

File: web/satellite/src/utils/localData.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    private static selectedProjectId: string = 'selectedProjectId';
+
+    public static getUserId(): string | null {
+        return localStorage.getItem(LocalData.userId);
+    }
+
+    public static setUserId(id: string): void {
+        localStorage.setItem(LocalData.userId, id);
+    }
+
```
Comment (NikolaiYurchenko) [inline web/satellite/src/utils/localData.ts]: remove userId from localStorage on logout

Comment (VitaliiShpital) [reply web/satellite/src/utils/localData.ts]: done

**Pull request merged**